tests/test_aea.py

```diff
@@ -72,13 +73,15 @@ def test_act():
         ledger_apis = LedgerApis({}, 'default')
         address = wallet.addresses['default']
         connections = [OEFLocalConnection(address, node)]
+        resources = Resources(str(Path(CUR_PATH, "data", "dummy_aea")))
```

Comment:
Could be replaced with `os.path.join(...)`. Returns str so doesn't require an extra action - to lead it to a str.

Reply:
If that's a common action (I've seen this before) I would request to upgrade `Path` with `to_str()` method.

Reply:
Thanks, I agree. But this is applied throughout the entire project. So I won't change it in this PR.